ReactApp/src/misc/Helpers.js

```diff
@@ -37,6 +37,22 @@ export function showBlockPicker() {
 	}
 }
 
+export function getPost() {
+	const currentPost = window?.initialPost;
+	if (currentPost) {
+		return {
+			id: currentPost.id,
+			title: { raw: decodeURIComponent(currentPost.title) },
+			content: { raw: decodeURIComponent(currentPost.content) },
+			type: currentPost?.type ?? 'post',
```

Comment:
Given the wrapping conditional, this optional chain is unnecessary, correct? Should we instead rely upon the OR operator? 
```suggestion
			type: currentPost.type || 'post',
```

Reply:
Updated in https://github.com/wordpress-mobile/GutenbergKit/pull/21/commits/64665da320dd26e6a8999539340b1d73794b1442 thanks!